modules/svneverever/__main__.py

```diff
@@ -255,12 +256,17 @@ def command_line():
 
     return args
 
+def login(realm, username, may_save):
+    username = six.input("Please enter your username: ")
+    password = getpass.getpass()
```

Comment:
I think we need some error handling in `login` as well, e.g. `KeyboardInterrupt` or `EOFError` during `getpass`